features/directory_structure.feature

```diff
@@ -36,10 +36,10 @@ Feature: Directory Structure
   `RSpec.describe` block with the `type: :controller` metadata:
 
   ```ruby
-  ​# spec/legacy/things_controller_spec.rb
+  # spec/legacy/things_controller_spec.rb
```

Comment:
Actually, 

![image](https://user-images.githubusercontent.com/6916/77215659-696e7f80-6b26-11ea-8523-0937de0e895d.png)

Reply:
It seems that the "weird character" is there for a reason, it tricks gherkin parser to ignore the comment because otherwise, the parser expects the free-form description to end after the comment. The original pull request doesn't explain nor highlight that it was done. I'll revert this change.
Tried newer `cucumber`/`gherkin` just to make sure it's not a bug in earlier versions.